src/app/test-record/test-section/edit/test-section-edit.component.html

```diff
@@ -36,7 +36,215 @@
+              >
+                Additional details for abandoning
+              </label>
+              <span id="additionalCommentsForAbandon" class="govuk-hint">
+                Notes about the reason for abandoning should be displayed here...
+              </span>
+              <textarea
+                class="govuk-textarea"
+                id="more-detail"
+                formControlName="additionalCommentsForAbandon"
+                rows="5"
+                aria-describedby="additionalCommentsForAbandon"
+                >{{ testType.additionalCommentsForAbandon }}</textarea
+              >
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div> -->
+
+  <div
+    *ngIf="testTypesApplicable.certificateApplicable[testType.testTypeId]"
+    class="govuk-grid-row govuk-!-margin-bottom-7"
+  >
+    <div class="govuk-grid-column-one-half">
+      <label class="govuk-label govuk-!-font-weight-bold">
+        Certificate number
+      </label>
+      <input
+        class="govuk-input"
+        id="test-certificateNumber"
+        formControlName="certificateNumber"
+        type="text"
+      />
+    </div>
+  </div>
+
+  <div
+    *ngIf="testTypesApplicable.expiryDateApplicable[testType.testTypeId]"
+    class="govuk-grid-row govuk-!-margin-bottom-3"
+  >
+    <div class="govuk-grid-column-three-quarters">
+      <label class="govuk-label govuk-!-font-weight-bold">
+        Expiry Date
+      </label>
+      <vtm-date-input
+        id="test-testExpiryDate"
+        ariaDescribedBy="testExpiryDate"
+        formControlName="testExpiryDate"
+        ngDefaultControl
+      >
+      </vtm-date-input>
+    </div>
+  </div>
+
+  <div
+    *ngIf="testTypesApplicable.anniversaryDateApplicable[testType.testTypeId]"
+    class="govuk-grid-row govuk-!-margin-bottom-3"
+  >
+    <div class="govuk-grid-column-three-quarters">
+      <label class="govuk-label govuk-!-font-weight-bold">
+        Anniversary Date
+      </label>
+      <vtm-date-input
+        id="test-testAnniversaryDate"
+        ariaDescribedBy="testAnniversaryDate"
+        formControlName="testAnniversaryDate"
+        ngDefaultControl
+      >
+      </vtm-date-input>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row govuk-!-margin-bottom-3">
+    <div class="govuk-grid-column-three-quarters">
+      <label class="govuk-label govuk-!-font-weight-bold">
+        Test date
+      </label>
+      <vtm-date-input
+        id="test-testTypeEndTimestampDate"
+        ariaDescribedBy="testTypeEndTimestampDate"
+        formControlName="testTypeEndTimestampDate"
+        ngDefaultControl
+      >
+      </vtm-date-input>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row govuk-!-margin-bottom-3">
+    <div class="govuk-grid-column-three-quarters">
+      <label class="govuk-label govuk-!-font-weight-bold">
+        Start time
+      </label>
+      <vtm-time-input
+        id="test-testTypeStartTimestamp"
```

Comment:
rename edit with test-edit.
We use test- for view screens and test-edit- for edit screens